code_challenges/code_wars/vowel_count.py

```diff
@@ -0,0 +1,7 @@
+# https://www.codewars.com/kata/54ff3102c1bad923760001f3
+
+
+def vowel_count(sentence):
+    vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
```

Comment:
This list is only used to check for inclusion, so you might consider a set like `{"a", "e", "i", "o", "u", "A", "E", "I", "O", "U"}`.  Since the list is short, it isn't sure to be more performant, but using a set still is a good pattern.